src/main/java/eventprocessing/fileservices/CSVFileService.java

```diff
@@ -13,20 +15,23 @@
 
 public class CSVFileService {
     private final String outputCsvFile;
-    private final Analyser analyser;
-    private int ESTIMATED_LINE_LENGTH = 70;
+    private final int ESTIMATED_LINE_LENGTH = 70;
+
+    static Logger logger = LogManager.getLogger(Main.class);
 
     public CSVFileService(String outputCsvFile) {
         this.outputCsvFile = outputCsvFile;
-        this.analyser = new Analyser();
     }
 
-    public void writeBucketDataToFile(Bucket bucketToWriteToFile) throws IOException {
+    public void write(Bucket bucketToWriteToFile) {
+        logger.info("Writing bucket with TimeRange " + bucketToWriteToFile.getTimeRange() + "and " + bucketToWriteToFile.getSqsResponses().size() + " responses.");
         CSVFormat csvFormat = getCsvFormat();
         StringBuffer stringBuffer = new StringBuffer(ESTIMATED_LINE_LENGTH * 2);
         try (CSVPrinter csvPrinter = new CSVPrinter(stringBuffer, csvFormat)) {
             writeBucketToStream(csvPrinter, bucketToWriteToFile);
             writeToFile(stringBuffer);
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
```

Comment:
This uses `System.out` instead of `logger`. Are you converting all the logging to log4j?